rules/windows/process_creation/proc_creation_win_pua_trufflehog.yml

```diff
@@ -0,0 +1,30 @@
+title: PUA - TruffleHog Execution
+id: 44030449-b0df-4c94-aae1-502359ab28ee
+status: experimental
+description: |
+    Detects execution of TruffleHog, a tool used to search for secrets in git repositories that could be used maliciously.
+    It was observed in the Shai-Hulud malware campaign targeting npm packages to steal sensitive information.
+references:
+    - https://github.com/trufflesecurity/trufflehog
+    - https://www.getsafety.com/blog-posts/shai-hulud-npm-attack
+author: Swachchhanda Shrawan Poudel (Nextron Systems)
+date: 2025-09-24
+tags:
+    - attack.discovery
+    - attack.t1083
+    - attack.t1552.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        Image|endswith: '\trufflehog.exe'
+    selection_cli:
+        CommandLine|contains|all:
+            - '--results=verified'
+            - '--json'
+            - 'filesystem'
```

Comment:
this command line argument signature looks to me more like a ET rule than a generic rule. Either we move the rule or write generic detection for the tool